test/test_scala_kernel.py

```diff
@@ -90,7 +90,7 @@ def test_init_magic(spylon_kernel):
 def test_init_magic_with_appname(spylon_kernel):
     code = dedent("""\
         %%init_spark
-        application_name = 'Dave'
+        launcher.conf.spark.app.name = 'Dave'
```

Comment:
I've been seeking an ungendered version of strawman. Draft? Suggestion? Starting point? None are as good.